puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_build_deb_dependency.yaml

```diff
@@ -23,14 +23,13 @@
           name: arch
           values:
           - x86
-          - armv8
       - axis:
           type: slave
           name: label
           values:
           - debian
     execution-strategy:
-      combination-filter: '(onlyos == "all" || onlyos == os) && !((version == "2.2" || version == "2.1" || version == "2.0" ) && os == "focal") && !((version == "2.2" || version == "2.1" || version == "nightly") && arch == "armv8")'
+      combination-filter: '(onlyos == "all" || onlyos == os) && !((version == "2.2" || version == "2.1" || version == "2.0" ) && os == "focal")'
```

Comment:
I think this also need to change to add 2.3 since we don't have focal yet?

Reply:
This PR is only about dropping ARM.